homework2/fastq_filtrator.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+# coding: utf-8
+def gc_filtering(read, gc_bounds=(0, 100)):
+    gc_num = len([GC for GC in read[1] if GC == "G" or GC == "C"])
```

Comment:
Не знаю задумывал ли ты так специально, но это по сути самый эффективный способ расчёта GC. Я бы лучше правда написал чуть по другому
```suggestion
    gc_num = sum(GC == "G" or GC == "C" for GC in read[1])
```